src/main/scala/elevate/rise/strategies/predicate.scala

```diff
@@ -14,80 +14,104 @@ object predicate {
       case l: Lambda => Success(l)
       case _         => Failure(isLambda)
     }
-    override def toString = "isLambda"
+    override def toString: String = "isLambda"
+  }
+
+  case object isDepLambda extends Strategy[Rise] {
+    def apply(e: Rise): RewriteResult[Rise] = e match {
+      case l: DepLambda[_] => Success(l)
+      case _               => Failure(isDepLambda)
+    }
+    override def toString: String = "isDepLambda"
+  }
+
+  case object isMakeArray extends Strategy[Rise] {
+    def apply(e: Rise): RewriteResult[Rise] = e match {
+      case m: MakeArray => Success(m)
+      case _            => Failure(isMakeArray)
```

Comment:
are you still using `scalafmt`? We should replace it with `scalastyle`, and I don't think we should align match cases like that.